web_src/js/components/PullRequestMergeForm.vue

```diff
@@ -39,6 +40,9 @@ export default {
   watch: {
     mergeStyle(val) {
       this.mergeStyleDetail = this.mergeForm.mergeStyles.find((e) => e.name === val);
+      for (const elem of document.getElementsByClassName(this.mergeForm.mergeCmdClass)) {
+        toggleElem(elem, elem.getAttribute('data-name') === val);
+      }
```

Comment:
It seems that `'mergeCmdClass': 'pull-merge-cmd',` is not needed, because the `data-name` is also specialized here.

Maybe the code could be simplified to:

```
      for (const elem of document.querySelectorAll('[data-pull-merge-style]')) {
        toggleElem(elem, elem.getAttribute('data-pull-merge-style') === val);
      }
```

(and remove the 'mergeCmdClass')